sdk/couchbase-core/src/mgmtx/mgmt.rs

```diff
@@ -221,369 +216,6 @@ impl<C: Client> Management<C> {
 
         parse_response_json(resp).await
     }
-
```

Comment:
Why are we deleting a bunch of other API code? If it's refactoring which helps this commit, then it would be a lot easier to review as two separate commits

Reply:
Ah, that's ok actually, makes sense